code/datums/wires/explosive.dm

```diff
@@ -11,6 +11,45 @@
 /datum/wires/explosive/proc/explode()
 	return
 
+/datum/wires/explosive/chem_grenade
+	holder_type = /obj/item/grenade/chem_grenade
+	randomize = TRUE
+	var/fingerprint
+	var/assembly

```

Comment:
assemblies already exist on the variable, just use that.

Reply:
if i do S.name it gives undefined var on explode() proc

Reply:
assemblies is a list

Reply:
what

Reply:
https://github.com/tgstation/tgstation/blob/40bef5a3aa05fc5d8dcee7b06efe7d8d4b8407ce/code/datums/wires/_wires.dm#L33

Reply:
that list is made of wires

Reply:
nani? it literally says it's the list of attached assemblies

Reply:
when i did assembly[1] it gave me "cyan"

Reply:
i'm too braindead to want to look into it